Keas.Mvc/Keas.Mvc.csproj

```diff
@@ -8,6 +8,7 @@
   </PropertyGroup>
   <ItemGroup>
     <PackageReference Include="AspNetCore.Security.CAS" Version="2.0.1" />
+    <PackageReference Include="Bootstrap.Datepicker" Version="1.8.0" />
```

Comment:
@jcubbage why are we installing this as a NUGET package?  Seems like it should be an NPM package like our other JS libraries (ex: bootstrap itself)